runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/getPotentialCapabilities/gpc001.c

```diff
@@ -131,34 +131,28 @@ Java_com_ibm_jvmti_tests_getPotentialCapabilities_gpc001_verifyOnLoadCapabilitie
 {
 	int availableCount = 0;
 	int unavailableCount = 0;
+	jboolean result = JNI_TRUE;
 	
 	getCapabilities(&initialCapabilities, &availableCount, &unavailableCount);
 
-	if (initialCapabilities.can_retransform_any_class == 0) {
+	if (0 == initialCapabilities.can_retransform_any_class) {
```

Comment:
If you're changing these unnecessarily, I'd prefer to see `!cap` instead of `0 == cap`

Reply:
I attempted to fix code format within the method modified for current PR.
In this case, the `rvalue` is moved to the left for equality comparison.
`initialCapabilities.can_retransform_any_class` is a `unsigned int` instead of `boolean` as per https://github.com/eclipse/openj9/blob/ba27f519d48a2a4b6dd811212ec106313b734052/runtime/include/jvmti.h.m4#L482.

Reply:
@DanHeidinga Do you have an opinion on this?

Reply:
Jason's made the suggested changes - let me know if you disagree.